firmware/application/freqman.cpp

```diff
@@ -103,51 +103,48 @@ std::vector<std::string> get_freqman_files() {
 };
 
 bool load_freqman_file(std::string& file_stem, freqman_db& db) {
-    return load_freqman_file_ex(file_stem, db, true, true, true);
+    return load_freqman_file_ex(file_stem, db, true, true, true, FREQMAN_MAX_PER_FILE);
 }
 
-bool load_freqman_file_ex(std::string& file_stem, freqman_db& db, bool load_freqs, bool load_ranges, bool load_hamradios) {
+bool load_freqman_file_ex(std::string& file_stem, freqman_db& db, bool load_freqs, bool load_ranges, bool load_hamradios, uint8_t limit) {
+    // these are not enough to really start with a new, clean, empty vector
```

Comment:
nit: looks like the comment refers to deleted lines?
"Swap with empty vector to ensure memory is immediately released."

Reply:
Deleted copy pasted comments and kept only the most significant